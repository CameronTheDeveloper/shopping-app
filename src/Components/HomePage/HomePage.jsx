import styles from "./HomePage.module.css";

function HomePage() {

    return (
        <main>
            <div>
                <h1>Welcome to Shopping App</h1>
                <h2>Shop in the comfort of your own home!</h2>
            </div>
        </main>
    );
}

export default HomePage;