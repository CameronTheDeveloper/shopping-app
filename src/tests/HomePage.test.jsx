import { render } from "@testing-library/react";
import HomePage from "../Components/HomePage/HomePage";

describe("HomePage component", () => {
    it("renders correct content", () => {
        const { asFragment } = render(<HomePage></HomePage>);
        expect(asFragment()).toMatchSnapshot();
    });
});
