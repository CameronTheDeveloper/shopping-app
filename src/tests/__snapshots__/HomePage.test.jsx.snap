// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomePage component > render correct content 1`] = `
<DocumentFragment>
  <h1>
    Welcome to Shopping App
  </h1>
  <h2>
    Shop in the comfort of your own home!
  </h2>
</DocumentFragment>
`;

exports[`HomePage component > renders correct content 1`] = `
<DocumentFragment>
  <h1>
    Welcome to Shopping App
  </h1>
  <h2>
    Shop in the comfort of your own home!
  </h2>
</DocumentFragment>
`;

exports[`HomePage component > renders correctly 1`] = `
<DocumentFragment>
  <h1>
    Welcome to Shopping App
  </h1>
  <h2>
    Shop in the comfort of your own home!
  </h2>
</DocumentFragment>
`;
